devicetypes/smartthings/springs-window-fashions-shade.src/springs-window-fashions-shade.groovy

```diff
@@ -266,4 +261,29 @@ def refresh() {
             zwave.switchMultilevelV1.switchMultilevelGet().format(),
             zwave.batteryV1.batteryGet().format()
     ], 1500)
+}
+
+def levelChangeFollowUp(expectedLevel) {
+    state.expectedValue = expectedLevel
+    state.levelChecks = 0
+    runIn(5, "checkLevel")
+}
+
+def checkLevelReport(value) {
+    if (state.expectedValue) {
+        if ((state.expectedValue == 99 && value > 99) ||
+                state.expectedValue == value) {
```

Comment:
I'm guessing that the `state.levelChecks` that is checked in `checkLevel` is what is protecting us in the event that we tell the shades to go to 60% and they stop at 61%?

Reply:
yeah, capped at 5 retries.

Do you think I should do some "close enough" if it's within like +/- 5?

Reply:
@greens I was wondering that. If you wanted to, I'd do +/- 2. I think I've seen the shades get pretty close. I know the iBlinds folks were saying that shades could take some time (I think up to 2 minutes?). You could also use `state.levelChecks` as a multiplier in the `runIn` timer. Something like `runIn(5 * (state.levelChecks + 1), ...)`, or some other computation to get you a few close values and then a few far out values.

Reply:
I'd probably do a check that allows +/- 1 or +/- 2. That will likely kill off a couple of the checks instead of running unneeded checks (further reducing cloud cycles as well as network traffic).